src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return PI * Math.pow(radius, 2);
```

Comment:
it's pefectly fine, but JFYI we mostly write x * x to take something to the power of two